src/styles/button.scss

```diff
@@ -0,0 +1,25 @@
+.buy {
+  justify-content: center;
+  box-sizing: border-box;
+  align-items: center;
+  text-transform: uppercase;
+  line-height: 16px;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  color: #fff;
+  text-decoration: none;
+  border-radius: 5px;
+  margin: 0 16px 16px;
+  padding: 12px 70px;
```

Comment:
The width is set to 166px and padding is set to 12px 70px. This could cause layout issues as the total width including padding exceeds the specified width, potentially causing overflow or unexpected layout behavior. Consider adjusting the width or padding to ensure consistency.